### ISSUE TITLE
404 not found when running apt-get update in build

### ISSUE THREAD

**Issue by justremember** [jruffet/docker-shogigui#1]
```
[+] Auto selected CPU arch : AVX2
[+] Building docker image with tag : shogigui0.0.7.23:yaneuraou6.00-AVX2

Sending build context to Docker daemon  381.4kB
Step 1/36 : ARG SHOGIGUI_VERSION=0.0.7.23
Step 2/36 : ARG YANEURAOU_VERSION=6.00
Step 3/36 : ARG YANEURAOU_TARGET_CPU=AVX2
Step 4/36 : ARG NPROC=4
Step 5/36 : ARG ELMO_GDRIVE_ID="1qhutTzaog4pHqh0OPAhJuf8mCwPAl5r7"
Step 6/36 : FROM ubuntu:19.10 AS build
 ---> 2f6c85efea61
Step 7/36 : LABEL app=shogigui
 ---> Using cache
 ---> f0b23c4977c4
Step 8/36 : LABEL stage=build
 ---> Using cache
 ---> 170b45c361ab
Step 9/36 : RUN apt-get update &&     apt-get install -y     clang     make     curl     unzip     p7zip-full     python3-pip     && rm -rf /var/lib/apt/lists/*
 ---> Running in 60f3ec84e7e8
Ign:1 http://security.ubuntu.com/ubuntu eoan-security InRelease
Ign:2 http://archive.ubuntu.com/ubuntu eoan InRelease
Err:3 http://security.ubuntu.com/ubuntu eoan-security Release
  404  Not Found [IP: 91.189.88.142 80]
Ign:4 http://archive.ubuntu.com/ubuntu eoan-updates InRelease
Ign:5 http://archive.ubuntu.com/ubuntu eoan-backports InRelease
Err:6 http://archive.ubuntu.com/ubuntu eoan Release
  404  Not Found [IP: 91.189.88.142 80]
Err:7 http://archive.ubuntu.com/ubuntu eoan-updates Release
  404  Not Found [IP: 91.189.88.142 80]
Err:8 http://archive.ubuntu.com/ubuntu eoan-backports Release
  404  Not Found [IP: 91.189.88.142 80]
Reading package lists...
E: The repository 'http://security.ubuntu.com/ubuntu eoan-security Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu eoan Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu eoan-updates Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu eoan-backports Release' does not have a Release file.
The command '/bin/sh -c apt-get update &&     apt-get install -y     clang     make     curl     unzip     p7zip-full     python3-pip     && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
```

**Comment by DerAndereJohannes** [comment]
This is a problem with Linux Mint users mentioned on the docker website ( https://docs.docker.com/engine/install/ubuntu/ )! To get around this (eg. for me using Mint 20.1 ulyssa), I had to change two things. Firstly, I had to change the Dockerfile ubuntu version from 19.10 to 20.04 (careful, there are two of these in the file) and I had to change inside the file at ```/etc/apt/sources.list.d/additional-repositories.list``` "**bionic**" to "**focal**" (ie. the ubuntu equivalent version) and this solves the issue!

Additional side note, on the line ```RUN mkdir /etc/mono/registr```, I had to add a -p flag (ie. change to ```RUN mkdir -p /etc/mono/registr```) since the script kept complaining that the folder already existed!

After this, the script worked perfectly!

**Comment by justremember** [comment]
Thanks I'll check it out when I get back to my linux laptop. Is the `/etc/apt/sources.list.d/additional-repositories.list` file in the docker image?

**Comment by DerAndereJohannes** [comment]
its the one on your own system! The file as mentioned contains a line that has to do with docker! I assume that docker gets its updated repositories from the link in the additional-repositories.list file!

**Comment by jruffet** [comment]
This issue has been fixed by 35066222379951e43836361eb20b2950f517620e
Upgraded ubuntu image version from eoan (EOL, hence the missing repositories) to impish
